src/screens/AccountUtxoBubbles/components/BubblePacking.tsx

```diff
@@ -56,14 +57,20 @@ export const BubblePacking = ({
 
         const text = data.value.toLocaleString() + ' sats';
 
+        // center the text inside the circle horizontally
         const getX = () => {
           const textDimensions = isSelected
             ? selectedFont?.measureText(data?.value ? text : '')
             : font?.measureText(data?.value ? text : '');
+          // TODO: find a better way to center the text horizontally
+          // "1.45" is just to make the text align properly in smaller Circle
```

Comment:
This 1.45 is strange, I don't understand it.  I was looking at skia api to see if there was anything to help with the calculation and didn't find anything yet.  I tried UTXO sizes down in single digits (5 sats, etc) and found that they would be off center, but using 1.5 instead of 1.45 seemed to correct it.

Reply:
Thank you Tom, much appreciated